src/cargo/sources/git/known_hosts.rs

```diff
@@ -588,7 +590,18 @@ impl KnownHost {
         }
         for pattern in self.patterns.split(',') {
             let pattern = pattern.to_lowercase();
-            // FIXME: support * and ? wildcards
+            let is_glob = is_glob_pattern(&pattern);
+
+            if is_glob {
+                let regex_pattern =
+                    regex::escape(&pattern.replace("*", ",,,,").replace("?", ";;;;"))
+                        .replace(",,,,", ".*")
+                        .replace(";;;;", ".");
```

Comment:
Is there a reason you are manually ensuring `*` and `?` get passed through `regex::escape` vs using `glob`?

Reply:
Oh, I somehow thought `glob` is only applicable for paths. Thank you for pointing this out!